core/new-gui/src/app/workspace/service/joint-ui/joint-ui.service.ts

```diff
@@ -135,125 +145,112 @@ export class JointUIService {
-      type: 'devs.TexeraModel',
-      markup:
-        `<g class="element-node">
-          <rect class="body" stroke-width="2" stroke="blue" rx="5px" ry="5px"></rect>
-          ${this.deleteButtonSVG}
-          <text></text>
-        </g>`
-    });
+  public static getCustomPortStyleAttrs(): joint.attributes.SVGAttributes {
+    const portStyleAttrs = {
+      '.port-body': {
+        fill: '#A0A0A0',
+        r: 5,
+        stroke: 'none'
+      },
+      '.port-label': {
+        display: 'none'
+      }
+    };
+    return portStyleAttrs;
+  }
+
+  /**
+   * This function creates a custom svg style for the operator.
+   * This function also make sthe delete button defined above to emit the delete event that will
+   *   be captured by JointJS paper using event name *element:delete*
+   *
+   * @param operatorDisplayName the name of the operator that will display on the UI
+   * @returns the custom attributes of the operator
+   */
+  public static getCustomOperatorStyleAttrs(operatorDisplayName: string): joint.shapes.devs.ModelSelectors {
+    const operatorStyleAttrs = {
+      'rect': { fill: '#FFFFFF', 'follow-scale': true, stroke: '#CFCFCF', 'stroke-width': '2' },
+      'text': {
+        text: operatorDisplayName, fill: 'black', 'font-size': '12px',
+        'ref-x': 0.5, 'ref-y': 0.5, ref: 'rect', 'y-alignment': 'middle', 'x-alignment': 'middle'
+      },
+      '.delete-button': {
+        x: 135, y: -20, cursor: 'pointer',
+        fill: '#D8656A', event: 'element:delete'
+      },
+    };
+    return operatorStyleAttrs;
   }
-}
 
-/**
- * This function changes the default svg of the operator ports.
- * It hides the port label that will display 'out/in' beside
- * the operators.
- *
- * @returns the custom attributes of the ports
- */
-export function getCustomPortStyleAttrs(): Object {
-  const portStyleAttrs = {
-    '.port-body': {
-      fill: '#A0A0A0',
-      r: 5,
-      stroke: 'none'
-    },
-    '.port-label': {
-      display: 'none'
-    }
-  };
-  return portStyleAttrs;
-}
 
-/**
- * This function creates a custom svg style for the operator. Also, this function
- * will make the delete button defined above to emit the delete event that will
- * be captured by JointJS.
- *
- * @param operatorDisplayName the name of the operator that will display on the UI
- * @returns the custom attributes of the operator
- */
-export function getCustomOperatorStyleAttrs(operatorDisplayName: string): Object {
-  const operatorStyleAttrs = {
-    'rect': { fill: '#FFFFFF', 'follow-scale': true, stroke: '#CFCFCF', 'stroke-width': '2' },
-    'text': {
-      text: operatorDisplayName, fill: 'black', 'font-size': '12px',
-      'ref-x': 0.5, 'ref-y': 0.5, ref: 'rect', 'y-alignment': 'middle', 'x-alignment': 'middle'
-    },
-    '.delete-button': {
-      x: 135, y: -20, cursor: 'pointer',
-      fill: '#D8656A', event: 'element:delete'
-    },
-  };
-  return operatorStyleAttrs;
 }
+
+
```

Comment:
Please remove these lines. Unnecessary changes to whitespace makes Git think that something has changed and basically two big block of functions show up on deleted and added side.

Reply:
Let me try to do that, hopefully git can be smarter to pick up the difference. The cause might not be the empty lines though. Originally these functions are just functions, they are outside of the class. Now these functions are static class functions, and they are inside the class. That should be the main reason.